cli/gardener_ci/productutil.py

```diff
@@ -195,19 +278,23 @@ def add_dependencies(
             c=component_name,
             v=component_version,
             f=descriptor_src_file
-        )
-        )
-
-    component_deps = component.dependencies()
-
-    for component_ref in component_dependencies:
-        component_deps.add_component_dependency(component_ref)
-    for image_dep in container_image_dependencies:
-        component_deps.add_container_image_dependency(image_dep)
-    for web_dep in web_dependencies:
-        component_deps.add_web_dependency(web_dep)
-    for generic_dep in generic_dependencies:
-        component_deps.add_generic_dependency(generic_dep)
+        ))
+
+    # maintain old behaviour
```

Comment:
maybe give a bit more context; e.g. "default value for backwards compatibility")?